reactDashboard/src/components/DataCharts/DataCharts.jsx

```diff
@@ -72,23 +78,23 @@ export function parseNhsCsvData(csvData) {
       y:
         totalCases5DayWindow === 0
           ? 0
-          : (positiveCases5DayWindow * 100) / totalCases5DayWindow,
+          : (positiveCases5DayWindow * 100) / totalCases5DayWindow
     });
     dailyCasesPoints.push({
       t: date,
-      y: cases,
+      y: cases
```

Comment:
Can you rerun your autoformat on these files please? :)

Reply:
I think this is resolved now